includes/wikia/services/WikiService.class.php

```diff
@@ -97,8 +102,42 @@ function() use ( $wiki, $useMaster, $excludeBots, $limit ) {
 		return $userIds;
 	}
 
-	private static function getAdminIdsFromDB( DatabaseBase $db, $excludeBots = false, $limit = null ) {
-		$conditions = [ 'ug_group' => self::ADMIN_GROUPS ];
+	/**
+	 * get list of wiki content/discussions moderator ids
+	 *
+	 * @param integer $wikiId - wiki Id (default: current wiki Id)
+	 * @param bool    $useMaster - flag that describes if we should use masted DB (default: false)
+	 * @param bool    $excludeBots - flag that describes if bots should be excluded from moderatorlist (default: false)
+	 * @param integer $limit - limit for the number of moderators
+	 *
+	 * @return array of $userIds
+	 */
+	public function getWikiModeratorIds( $wikiId = 0, $useMaster = false, $excludeBots = false, $limit = null ) {
+		$wikiId = empty( $wikiId ) ? $this->wg->CityId : $wikiId ;
```

Comment:
I will go with the styling suggestion and ?? operator. As for global, this file uses $this->wg and I do not want to change it in the scope of this work.